src/boundingmesh/CMakeLists.txt

```diff
@@ -4,9 +4,16 @@ find_package(Eigen 3 REQUIRED)
 find_package(Coin)
 find_package(CGAL)
 find_package(GMP)
+find_package(assimp)
 
 set(BOUNDINGMESH_INCLUDE_DIRS ${EIGEN_INCLUDE_DIRS})
 
+if (assimp_FOUND)
```

Comment:
drop this section, this should not be needed
if yes, perhaps there is not cmake find script for assimp, in that case please write one in https://github.com/gaschler/bounding-mesh/tree/master/Modules